### DISCUSSION TITLE
Copilot and Csla

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3582] [Questions] [answered]
Hi All,
I was reading Rocky's blog post [The Programmer Gap](https://blog.lhotka.net/2023/11/19/The-Programmer-Gap).
I haven't used Copilot yet but was wondering how useful will it be for Csla developers?
Kind Regards

**Comment by rockfordlhotka** [comment, marked as answer]
I find it useful, in that it is pretty good about replacing the use of snippets to create properties, and goes beyond that by also suggesting entire class structures.

It isn't perfect, but absolutely boosts productivity over typing everything by hand.

**Reply by Chicagoan2016** [reply]
@rockfordlhotka  Thank you Sir.
